### PR TITLE
Update for work new methods for simplify 

### PR DESCRIPTION
**Pull request by TomashGombosh**
Added testNG suite, test lister. Update pom file update Login page, n…w Utils for random data generation

### CODE REVIEW

File: src/test/java/com/blackNecoProject/testng/commons/Utils.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ */
+public class Utils {
+
+
+    /**
+     * Waits for a moment.
+     *
+     * @param millis how many milliseconds to sleep.
+     */
+    public static void sleep(long millis){
```
Comment (KristianGombosh) [inline src/test/java/com/blackNecoProject/testng/commons/Utils.java:20]: for why this class?

Comment (TomashGombosh) [reply src/test/java/com/blackNecoProject/testng/commons/Utils.java]: Waits for some action before download or upload file to web site

Comment (TomashGombosh) [reply src/test/java/com/blackNecoProject/testng/commons/Utils.java:20]: for waiting uploading or downloading some element from website

**Review by KristianGombosh — APPROVED** [review]
Well done

**Pull request merged**